Kickstarter-iOS/Views/Cells/DiscoveryExpandedSelectableRow.swift

```diff
@@ -12,7 +12,7 @@ internal final class DiscoveryExpandedSelectableRowCell: UITableViewCell, ValueC
 
   internal func configureWith(value: (row: SelectableRow, categoryId: Int?)) {
     if let category = value.row.params.category, category.isRoot {
-      self.filterTitleLabel.text = RootCategory(categoryId: category.id).allProjectsString()
+      self.filterTitleLabel.text = RootCategory(categoryId: category.intID ?? -1).allProjectsString()
```

Comment:
`-1`?

Reply:
-1 is the id for unrecognized category. I've explained with more detail some comments below.